.github/workflows/tests.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Tests'
+
+on: [push]
```

Comment:
We could but that on pull requests only as we won't be watching for push

Reply:
When a PR is updated, does it trigger the hook with `pull_requests: opened/reopened` only?

Reply:
Do you know the correct syntax to specify that the job needs to be run on every pull request?